documentation/src/main/markdown/packagemgrs/docker/releasenotes.md

```diff
@@ -1,5 +1,11 @@
 # [docker_inspector_name] Release notes
 
+## Version 10.1.2
```

Comment:
This version does not match the version in the currentreleasenotes.md

Reply:
Good catch. I see the last version noted in the related build file is 10.1.1-SNAPSHOT so it could be that the docker release notes is in need of a change there to 10.1.1?